rpc/flipt/operators.go

```diff
@@ -74,4 +74,12 @@ var (
 		OpPresent:    {},
 		OpNotPresent: {},
 	}
+	EntityIdOperators = map[string]struct{}{
+		OpEQ:         {},
+		OpNEQ:        {},
+		OpEmpty:      {},
+		OpNotEmpty:   {},
+		OpIsOneOf:    {},
+		OpIsNotOneOf: {},
+	}
```

Comment:
Still wondering if we should support all of these operators?

Reply:
yeah i guess `empty` and `notempty` dont make sense as its required

Reply:
@markphelps Done!